cmsdb/campaigns/run3_2023_preBPix_nano_v12/data.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+
+"""
+CMS datasets from the 2023 preBPix data-taking campaign
+"""
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run3_2023_preBPix_nano_v12 import campaign_run3_2023_preBPix_nano_v12 as cpn
+
+# Add eras A and B also?
+
+#
+# Muon
+#
+cpn.add_dataset(
+    name="data_mu_c",
+    id=14786982,
+    is_data=True,
+    processes=[procs.data_mu],
+    keys=[
+        "/Muon0/Run2023C-22Sep2023_v4-v1/NANOAOD",  # noqa
+        "/Muon1/Run2023C-22Sep2023_v4-v1/NANOAOD",  # noqa
+        "/Muon1/Run2023C-22Sep2023_v4-v2/NANOAOD",  # noqa
+    ],
+    n_files=165 + 131 + 108,
+    n_events=138943783 + 101615754 + 138912262,
+    aux={
+        "era": "C",
+    },
```

Comment:
Are the two `Muon1` datasets really orthogonal datasets? I would assume that `v1` is a subset of events from `v2`

```suggestion
    keys=[
        "/Muon0/Run2023C-22Sep2023_v4-v1/NANOAOD",  # noqa
        "/Muon1/Run2023C-22Sep2023_v4-v2/NANOAOD",  # noqa
    ],
    n_files=165 + 108,
    n_events=138943783 + 138912262,
```